src/pages/MyPage/History/History.js

```diff
@@ -0,0 +1,131 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import HistoryTable from './HistoryTable';
+import HistoryHeader from './HistoryHeader';
+import PaymentSummaryList from './PaymentSummaryList';
+
+function History() {
+  const [investLog, setInvestLog] = useState([]);
+  const [search, setSearch] = useState('');
+  const [filterValue, setFilterValue] = useState('전체');
+
+  useEffect(() => {
+    fetch('/data/history.json')
+      .then(res => res.json())
+      .then(res => setInvestLog(res.results));
+  }, []);
+
+  const filterHandler = e => {
+    setFilterValue(e.target.value);
+  };
+
+  const handleChange = e => {
+    setSearch(e.target.value);
+  };
+
+  const filteredList = investLog.filter(product => {
+    return (
+      product.item.includes(search) &&
+      (filterValue !== '전체' ? product.status === filterValue : product.status)
+    );
+  });
+
+  return (
+    <HistoryWrapper>
+      <div>
+        <SearchBar onChange={handleChange}>
+          <i class="fas fa-search" />
+          <input
+            type="text"
+            placeholder="상품호수 또는 상품명을 검색해보세요"
+          />
+        </SearchBar>
+        <HistoryMain>
+          <HistoryHeader filterHandler={filterHandler} />
+          <PaymentSummary>
+            <PaymentSummaryList
+              arr={filteredList}
+              title={'투자금액'}
+              item={'invest'}
+            />
+            <PaymentSummaryList
+              arr={filteredList}
+              title={'지급완료 원금'}
+              item={'principal'}
+            />
+            <PaymentSummaryList
+              arr={filteredList}
+              title={'지급완료 이자'}
```

Comment:
```suggestion
              title='지급완료 이자'
```
string은 중괄호 안써도 됩니다!